src/main/java/com/minecolonies/util/LanguageHandler.java

```diff
@@ -62,8 +63,10 @@ public static String getString(String key)
      * @param defaultValue the value to return if no key is found
      * @return Localized string
      */
+    @SuppressWarnings("deprecation")
```

Comment:
In this case it might do but it won't give us a string back. It is a textcomponent to send to cleint chat messages for example. So idk if we can use that...